english/amnesia/amnesia.md

```diff
@@ -4,5 +4,5 @@ I try some tea, it gets me nowhere.
 It's probably because the marshmellows where filled with horse hooves
 It's definitely throwing off my groove:(
 Then I open my mac and log on to udacity :)
-
```

Comment:
Options with links are supposed to be visually separated from the rest of the story with a blank line. Could you leave the blank line there and add your sentence above it please?